src/capi/azext_capi/custom.py

```diff
@@ -618,7 +618,7 @@ def install_cni(cmd, cluster_name, workload_cfg, windows, args):
         chart_name="calico",
         chart="projectcalico/tigera-operator",
         values_file=values_file,
-        args=["--namespace", "tigera-operator", "--create-namespace"]
+        args=["--namespace", "tigera-operator", "--create-namespace", "--version",  os.environ.get("CALICO_VERSION", "v3.25.1")]
```

Comment:
I think we need to update line 642 below to also use `release-1.9` for the windows calico deployments

Reply:
And should we do something like 
 
```bash
kubectl set image -n calico-system daemonSet/calico-node-windows install-cni sigwindowstools/calico-install:${CALICO_VERSION}-hostprocess
kubectl set image -n calico-system daemonSet/calico-node-windows calico-node-startup sigwindowstools/calico-node:${CALICO_VERSION}-hostprocess
kubectl set image -n calico-system daemonSet/calico-node-windows calico-node-felix sigwindowstools/calico-node:${CALICO_VERSION}-hostprocess
```
in `install_cni_windows` to make sure we get the same version?